dashboard/src/actions/kube.tsx

```diff
@@ -44,6 +49,27 @@ export const closeWatchResource = createAction("CLOSE_WATCH_RESOURCE", resolve =
   return (ref: ResourceRef) => resolve(ref);
 });
 
+// requestResources takes a ResourceRef[] and subscribes to an observable to
+// process the responses as they arrive.
+export const requestResources = createAction("REQUEST_RESOURCES", resolve => {
```

Comment:
So, it is in this _request_ event that we also perform the actual subscription. Other actions (which are not meant to return any msg-subscription thing) split the _request_ and _receive_ actions, but here, given that the concept of requesting resources implies requesting a subscription, maybe makes sense to leave it this way.